implementation/src/main/java/io/smallrye/metrics/legacyapi/HistogramAdapter.java

```diff
@@ -42,6 +49,15 @@ public HistogramAdapter register(MpMetadata metadata, MetricDescriptor metricInf
 
         if (globalCompositeSummary == null || metadata.cleanDirtyMetadata()) {
 
+            MetricPercentileConfiguration percentilesConfig = MetricsConfigurationManager.getInstance()
+                    .getPercentilesConfiguration(metadata.getName());
+
+            HistogramBucketConfiguration bucketsConfig = MetricsConfigurationManager.getInstance()
+                    .getHistogrmBucketConfiguration(metadata.getName());
```

Comment:
Typo in `getHistogrmBucketConfiguration`?  Should be `getHistogramBucketConfiguration`

Reply:
fixed